projects/module-1/ideabox-group.md

```diff
@@ -203,110 +194,248 @@ As a user,
+This project has 3 evaluated concepts:
+
+- Workflow & Professional Habits
+- Comp Recreation with HTML & CSS
+- JavaScript
+
+And evaluation of these concepts is assisted through the final graded section:
+- Functionality (we can't properly assess the 3 main concepts if there is not enough work to grade)
+
+---
+
+### Workflow & Professional Habits
+
+This project gives the team room to practice the following skills:  
+- Collaboration
+- Git
+- GitHub
+
+Competency & understanding of these concepts can be demonstrated in many ways. **The following examples are not checklists to complete!** They are illustrations to guide your team as you develop this project and continue your learning.
+
+<section class="answer">
+### Collaboration competency examples
+
+- The team's norming document is taken seriously, with thoughtful reflections from all team members regarding their skills, learning goals, work styles, etc. 
+- The team plans and executes an agreed upon process for working, troubleshooting, and handling conflicts.
+- Each member of the team reflects meaningfully on the collaboration and can point to specific behaviors they want to continue, and those they want to modify for future collaboration
+- The work is distributed equitably; hopefully commit/code contributions are roughly even for each team member
+
+WOW can look like:
+
+- The team's norming document is regularly revisted and revised as better perspective is gained
+- When conflicts arise, the team works together to find ways to re-focus on individual and group learning objectives
+</section>
+
+Note: ⬆️this⬆️ kind of collaboration can be difficult to assess, but know that this is a crucial and vital part of your learning and marketability as a developer! 
+
+<section class="answer">
+### Git competency examples
+
+- Team understands the importance & utility of documenting a single change with every commit; changesets are small
+- Commit messages are descriptive and concise
+- The team keeps the main branch clean by using branches for new, unreviewed code
+
+WOW can look like:
+
+- The team's commit messages clearly tell the story of how the project evolved over time
+- Commits are atomic, documenting a single changeset (such as a new function being created, or a function being updated).
+</section>
+
+<section class="answer">
+### GitHub competency examples
+
+- Team understands the point of the PR process, making strong use of code reviews & pushing up requested changes before merging
+- PRs have clear, thorough descriptions, such that a new person joining the codebase can clearly understand the proposed changes and evolution of the codebase
+- The team understands that the README is an opportunity to document their work and growth
+- The team understands that the README is a new contributor or potential employer's first glimpse into the project, and includes thorough setup instructions, context, reflections, and other pertinent information
```

Comment:
Could we update these two lines to be more 'action' based rather than 'understanding' based?

For example:  "The team uses the README as..."

Reply:
Love this, definitely will do!